pkg/controller/manage/manage_controller.go

```diff
@@ -228,12 +236,12 @@ func enableKmeshManage(pod *corev1.Pod, queue workqueue.RateLimitingInterface, s
 		log.Errorf("failed to enable Kmesh manage")
 		return
 	}
-	queue.AddRateLimited(QueueItem{podName: pod.Name, podNs: pod.Namespace, action: ActionAddAnnotation})
-	_ = linkXdp(nspath, xdpProgFd, mode)
+	kmc.queue.AddRateLimited(QueueItem{podName: pod.Name, podNs: pod.Namespace, action: ActionAddAnnotation})
+	_ = linkXdp(nspath, kmc.xdpProgFd, kmc.mode)
```

Comment:
why handle this error？

Reply:
You mean why NOT handle this error? Because error is handled in `linkXdp`